cloud_images/centos7/install_prereqs.sh

```diff
@@ -33,12 +33,23 @@ Type=oneshot
 ExecStart=/usr/local/sbin/dcos_vol_setup.sh /dev/xvde /var/lib/mesos
 ExecStart=/usr/local/sbin/dcos_vol_setup.sh /dev/xvdf /var/lib/docker
 ExecStart=/usr/local/sbin/dcos_vol_setup.sh /dev/xvdg /dcos/volume0
+ExecStart=/usr/local/sbin/dcos_vol_setup.sh /dev/xvdh /var/log
 
 [Install]
 WantedBy=local-fs.target
 EOF
 systemctl enable dcos_vol_setup
 
+echo ">>> Disable rsyslog"
+systemctl disable rsyslog
+
+echo ">>> Set journald limits"
+mkdir -p /etc/systemd/journald.conf.d/
+echo -e "[Journal]\nRateLimitBurst=15000\nRateLimitInterval=30s\nSystemMaxUse=10G" > /etc/systemd/journald.conf.d/dcos-el7.conf
```

Comment:
why don't we just use / get this from the snippet we already always ship with DC/OS? Why do we need a special snippet here different than what we're shipping on all the other platforms?

https://github.com/dcos/dcos/blob/master/gen/cloud-config.yaml#L12

I really don't want different forms of our templates to have slightly different behaviors.